### ISSUE TITLE
coc vetur - Vetur 1149 - File name X differs from Y only in casing error

### ISSUE THREAD

**Issue by UmrenTV** [neoclide/coc-vetur#71]
I just installed the other day coc-vetur and am trying my working enviroment, and it seems that this error is consistant with couple of my projects but not across all imports. Here is a picture:

![image](https://user-images.githubusercontent.com/20478714/115414527-6563d400-a1f6-11eb-8eb5-5f2e0115ffaf.png)

I am sure that the import is correct, since it's already built for production and I am working on it as we speak without any errors, but these ones are kinda annoying.
Any way to turn off the casing errors, or to ignore them? (which I know it's not effective on the long run, but it triggers me when I have errors while working) :D

Update: If you look closely, the difference it complains about is in "Coding" vs "coding" in the path, but no idea how to change that, since I don't use absolute paths, but rather relative, so no idea how to fix it.
Also not all of the imports are complaining about this, it's 6 out of 9 total.

**Comment by chemzqm** [comment]
You should fix those issues since some platform is case sensitive

**Comment by UmrenTV** [comment]
> You should fix those issues since some platform is case sensitive

Fix those issues how? I am using relative path. I don't give it absolute path so I can fix it in my typing...
If I was doing manual import with import X from 'C:/x/y/z' i can do it, but since I use project relative paths with '../../component.vue' I don't know how can I fix it, since some imports (which are written the same way, are complaining about it, and some are not. 

Any input on how? :)

I worked on this project originally in VScode, and it didn't complained there anything, but in neovim it does, since I switched to it 2 weeks ago, I am still pretty new to it, and I had to go back to the project and do some updates on it.

Is there a way to ignore that warning from vetur, since the project works, and builds perfectly, and it's currently on production without any issues? I am sure it's because of the plugin or something :/ If there is some settings i can so in my coc-vetur like ignore casing, would be super great. Thanks
